submissions/kasionio/friends-app/index.js

```diff
@@ -0,0 +1,136 @@
+const cardsContainer = document.createDocumentFragment();
+const userCards = document.getElementById('user-cards');
+const API_URL = 'https://randomuser.me/api/?results=10&inc=name,picture,gender,location,dob,cell,email&noinfo';
+const NUMBER_OF_FETCH_RETRIES = 5;
+const MIN_AGE = 18;
+const MAX_AGE = 99;
+
+const fetchUsersData = async (url, retries) => {
+	try {
+		let response = await fetch(url);
+		let usersData = await response.json();
+		return usersData.results;
+	} catch (err) {
+		if (retries > 0) throw err;
+		return await fetch_retry(url, retries - 1);
```

Comment:
`fetch_retry`? Why you have used `snake_case`?
And where this function is declared?

Reply:
Indian-StackOverflow naming style